src/libraries/intellisense.targets

```diff
@@ -0,0 +1,76 @@
+<Project>
```

Comment:
As I mentioned previously this needs a comment explaining why we need this and a link to where we copied from